tensorflow/python/ops/image_ops_impl.py

```diff
@@ -412,9 +416,17 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
   image = ops.convert_to_tensor(image, name='image')
 
   assert_ops = []
-  assert_ops += _Check3DImage(image, require_static=False)
+  assert_ops += _CheckAtLeast3DImage(image, require_static=False)
+
+  is_batch = True
+  if image.get_shape().ndims == 3:
+    is_batch = False
+    image = array_ops.expand_dims(image, 0)
+  elif image.get_shape().ndims != 4:
+    raise ValueError('\'image\' must have either 3 or 4 dimensions.')
+
+  num_channels, height, width, depth = image.get_shape().as_list()
```

Comment:
this seems like it might be problematic for dynamic sizes. Perhaps I should add a `_ImageDimensions` that works in higher dimensions?